proto/src/server.rs

```diff
@@ -230,54 +253,93 @@ impl<D: indradb::Datastore + Send + Sync + 'static> crate::indra_db_server::Indr
         request: Request<crate::VertexQuery>,
     ) -> Result<Response<Self::GetVerticesStream>, Status> {
         let datastore = self.datastore.clone();
+
         let q: indradb::VertexQuery = map_conversion_result(request.into_inner().try_into())?;
         let (tx, rx) = mpsc::channel(CHANNEL_CAPACITY);
-        tokio::spawn(async move {
-            send(tx, datastore.get_vertices(q)).await;
+        Handle::current().spawn(async move {
```

Comment:
`tokio::spawn` should still be OK to call

Reply:
done

Reply:
marked resolved.